samples/net/zoap_server/prj.conf

```diff
@@ -9,3 +9,4 @@ CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_ZOAP=y
 CONFIG_NET_APP_SETTINGS=y
 CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
```

Comment:
can we set the default to 5 or any safe value and remove the need for guessing the number in many of the same we have?
Right now there are 38 places where this config has different values:

gg NET_IF_MCAST_IPV6_ADDR_COUNT samples/ tests/ | wc -l
38